.github/CODEOWNERS

```diff
@@ -130,13 +130,13 @@
 /sdk/digitaltwins/ @sjiherzig @johngallardo @olivakar
 
 # PRLabel: %Event Grid
-# AzureSdkOwners: @sarangan12 
+# AzureSdkOwners: @sarangan12
 # ServiceLabel: %Event Grid
 /sdk/eventgrid/ @sarangan12
 
 # PRLabel: %Event Hubs
 # ServiceLabel: %Event Hubs
-# AzureSdkOwners: @deyaaeldeen @jeremymeng 
+# AzureSdkOwners: @deyaaeldeen @jeremymeng
```

Comment:
You can't do it all in one block like this.  You'll need to break into two:

```text
# PRLabel: %Event Hubs
/sdk/eventhub/ @deyaaeldeen @jeremymeng @HarshaNalluru

# ServiceLabel: %Event Hubs
# AzureSdkOwners: @deyaaeldeen @jeremymeng
# ServiceOwners: @aschhabria
```

Reply:
This pattern occurs multiple times in existing file.  I undid the formatting change and will follow up with another PR.

Reply:
As a general recommendation, I'd always break into PR and support blocks.   The implicit relationship for paths to support doesn't always do what we think that it does.   For example, it is always assumed that the owners are service team members when they come from the PR path.

Reply:
addressing in another PR.